app/src/features/surveys/observations/sampling-sites/edit/components/SampleSiteEditForm.tsx

```diff
@@ -75,49 +58,65 @@ const SampleSiteEditForm: React.FC<ISampleSiteEditForm> = (props) => {
           <Paper
             elevation={0}
             sx={{
-              p: 5
+              p: 5,
+              '& hr': {
+                height: '1px',
+                mt: 5,
+                mb: 5
+              }
             }}>
             <HorizontalSplitFormComponent
               title="General Information"
               summary="Specify the name and description for this sampling site"
               component={<SampleSiteGeneralInformationForm />}></HorizontalSplitFormComponent>
 
-            <Divider className={classes.sectionDivider} />
+            <Divider />
 
             <HorizontalSplitFormComponent
               title="Site Location"
               summary="Import or draw sampling site locations used for this survey."
               component={<SurveySamplingSiteEditForm />}></HorizontalSplitFormComponent>
 
-            <Divider className={classes.sectionDivider} />
+            <Divider />
 
             <HorizontalSplitFormComponent
               title="Sampling Methods"
               summary="Specify sampling methods that were used to collect data."
               component={<SampleMethodEditForm name={'sampleSite.methods'} />}></HorizontalSplitFormComponent>
 
-            <Divider className={classes.sectionDivider} />
+            <Divider />
 
             <Box display="flex" justifyContent="flex-end">
-              <LoadingButton
-                type="submit"
-                variant="contained"
-                color="primary"
-                loading={props.isSubmitting}
-                onClick={() => {
-                  props.formikRef.current?.submitForm();
-                }}
-                className={classes.actionButton}>
-                Save and Exit
-              </LoadingButton>
-              <Button
-                variant="outlined"
-                color="primary"
-                component={RouterLink}
-                to={`/admin/projects/${surveyContext.projectId}/surveys/${surveyContext.surveyId}/observations`}
-                className={classes.actionButton}>
-                Cancel
-              </Button>
+              <Box
+                sx={{
+                  '& button': {
+                    minWidth: '6rem'
+                  },
+                  '& button + button': {
+                    ml: 1
+                  }
+                }}>
+                <LoadingButton
+                  type="submit"
+                  variant="contained"
+                  color="primary"
+                  loading={props.isSubmitting}
+                  onClick={() => {
+                    props.formikRef.current?.submitForm();
```

Comment:
Should be able to just use the formik context here? Since this component is a child of Formik

Reply:
Good catch, changes.